examples/nlp/language_modeling/conf/megatron_bert_config.yaml

```diff
@@ -132,8 +132,8 @@ model:
     splits_string: 900,50,50
     seq_length: ${model.encoder_seq_length}
     skip_warmup: True
-    num_workers: 0
-    dataloader_type: single # cyclic
+    num_workers: 1
```

Comment:
Is this a fixed value ?

Reply:
I will put the config back to they way it was but add an LDDL option in the comment next to dataloader type.  num workers is the same as in normal pytorch it is a dataloader argument https://github.com/NVIDIA/NeMo/pull/5977/files#diff-d496e911eb06fca3128061957d326a83ba0a603ca8482ad292aee3851f49bf61R515